<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="addWatingButton">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>110</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>C:/</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ChoseFolderButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Chose folder</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="addWatingLine">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>110</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="CheckButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>40</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
       <widget class="QListWidget" name="watingListWidget">
        <property name="geometry">
         <rect>
          <x>455</x>
          <y>160</y>
          <width>241</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="testCaseTable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>111</y>
          <width>391</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="openTestcaseLineEdit">
              <property name="text">
               <string>Chose file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ChoseTestCaseButton">
              <property name="text">
               <string>Chose testcase</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
